apps/mobile/src/features/activity/activity-list.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+
+import { ViewMode } from '@/shared/types';
+// NOTE: '/src' is necessary to fix an import issue with FlashList
```

Comment:
If I don't point to `@shopify/flash-list/src` `FlashList` doesn't work and crashes the APP. 

If I do point there it works but gives a ton of type errors. I need to spend some extra time investigating. Maybe someone else has ideas?

Reply:
I spent a few hours on this and couldn't solve it yet. [I opened a bug on their repo](https://github.com/Shopify/flash-list/issues/1661 )

I decided to just replace `FlashList` with native `FlatList` instead for now as it seems to give most of the improvements we want for now. 

I opened [a draft PR](https://github.com/leather-io/mono/pull/1184) which shows the problem, I'll keep a more isolated demo of FlashList there in case we revisit

Reply:
I got this solved with help from @tigranpetrossian . Thanks a lot!